streamly-process.cabal

```diff
@@ -100,4 +100,4 @@ test-suite Test.System.Process
     , hspec             >= 2.0   && < 3
     , process           >= 1.0   && < 1.7
     , QuickCheck        >= 2.10  && < 2.15
-    , streamly          >= 0.7.1 && < 0.8
+    , streamly          >= 0.8   && < 0.9
```

Comment:
We are depending on the internal imports. Do you want to make the upper bound inclusive of the latest working minor release? 0.8.0 in this case.